DSharpPlus.CommandsNext/CommandsNextExtension.cs

```diff
@@ -489,19 +489,14 @@ private void RegisterCommands(Type t, CommandGroupBuilder? currentParent, IEnume
                         foreach (var chk in inheritedChecks)
                             groupBuilder.WithExecutionCheck(chk);
 
-                        if (ti.DeclaredMethods.Any(x => x.IsCommandCandidate(out _) && x.GetCustomAttribute<GroupCommandAttribute>() != null))
-                        {
-                            var groupCommandOverloads =
-                                ti.DeclaredMethods.Where(x => x.IsCommandCandidate(out _) && x.GetCustomAttribute<GroupCommandAttribute>() != null);
-
-                            // overloads don't work for group commands, so throw on startup instead of at runtime.
-                            if(groupCommandOverloads.Count() > 1)
-                            {
-                                throw new InvalidOverloadException($"Commands marked with [{nameof(GroupCommandAttribute)}] cannot be overloaded", groupCommandOverloads.First());
-                            }
+                        var gcan = ti.DeclaredMethods.Where(x => x.IsCommandCandidate(out _) && x.GetCustomAttribute<GroupCommandAttribute>() != null).ToArray();
```

Comment:
"So long as we don't name it gcan" - Exa

Meanwhile exa's commit:

Reply:
i got distracted fixing syntax. name fixed too, now